packages/client/wallets/aa/src/api/BaseCrossmintService.ts

```diff
@@ -40,12 +41,14 @@ export abstract class BaseCrossmintService extends LoggerWrapper {
                 const { body, method } = options;
 
                 let response: Response;
-
                 try {
                     response = await fetch(url, {
                         body,
                         method,
-                        headers: this.crossmintAPIHeaders,
+                        headers: {
+                            ...this.crossmintAPIHeaders,
+                            ...(authToken != null && { authorization: `bearer ${authToken}` }),
```

Comment:
nit: It's my understanding that the convention is sending `Bearer` with capital `B`

Reply:
fixed, ty for catching